android/fastlane/Fastfile

```
@@ -57,15 +56,20 @@ platform :android do
       skip_upload_images: true,
       skip_upload_screenshots: true,
     )
-    # TODO: uncomment this when ready
-    # internal_app_url = upload_to_play_store_internal_app_sharing(
-    #   aab: "../build/app/outputs/bundle/release/app-release.aab",
-    # )
+    begin
+      internal_app_url = upload_to_play_store_internal_app_sharing(
+        aab: "../build/app/outputs/bundle/release/app-release.aab",
```

RuboCop (1.81.1)

[convention] Avoid comma after the last parameter of a method call.

(Style/TrailingCommaInArguments)